src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -47,6 +61,10 @@ contract BLSRegistryCoordinatorWithIndices is Initializable, IBLSRegistryCoordin
     mapping(address => Operator) internal _operators;
     /// @notice the dynamic-length array of the registries this coordinator is coordinating
     address[] public registries;
+    /// @notice the address of the entity allowed to sign off on operators getting kicked out of the AVS during registration
+    address public churner;
+    /// @notice the nonce of the churner used in EIP-712 signatures
+    uint256 public churnerNonce;
```

Comment:
for naming: the description suggests that this entity is really a `churnApprover` or similar.

am also thinking that 'salts' may make more sense than nonces here, but we can debate a bit -- the intended usage is important to consider

Reply:
Changed to churn approver, would prefer to keep the nonce logic for now since we expect this to not have many race conditions

Reply:
agreed offline that salt is more flexible here / nonce is not needed

Reply:
added